spec/features/audit/allocation/allocate_spec.rb

```diff
@@ -126,36 +126,83 @@
     let!(:books) do
       create_list(
         :content_item,
-        20,
+        60,
         primary_publishing_organisation: novelists,
       )
     end
 
     let!(:paintings) do
       create_list(
         :content_item,
-        20,
+        60,
```

Comment:
Why have we incremented to 60 the number of content items?

Reply:
We previously updated our pages to have 100 content items per page, but didn't update this test to span multiple pages (as it used to). This change creates more items so that it spans multiple pages.

Reply:
I have added: https://github.com/alphagov/content-performance-manager/pull/402/commits/7a665d3459b1e21e7afbd28989dcc21db90c252c, as a temporary commit with some `draft` code to move the logic into the `FindContent` object.
It will also allow us to test this method in isolation without having a feature test.

We still need to sort out the pagination issue (we are 100 set up in many places), but this would be out of the scope of this PR. 

Any thoughts?

Reply:
I've re-worked c58412c50204ef5745b2e5fa4743437801a2a7ec incorporating your suggestions. That's allowed me to move the test coverage down the pyramid. The testing is a bit clunky due to the behaviour that is contained within `Content::Query` but I've left that for another day!